hathor/verification/block_verifier.py

```diff
@@ -42,9 +42,38 @@ def verify(self, block: Block) -> None:
             # TODO do genesis validation
             return
 
-        block.verify_without_storage()
+        self.verify_without_storage(block)
 
         # (1) and (4)
-        block.verify_parents()
+        self.verify_parents(block)
+
+        self.verify_height(block)
+
+    def verify_without_storage(self, block: Block) -> None:
+        """ Run all verifications that do not need a storage.
+        """
+        block.verify_without_storage()
 
+    @staticmethod
+    def verify_height(block: Block) -> None:
```

Comment:
Why is it a static method?

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/797/commits/105e0598363e0125e9f6112ec72b213889790b61

Reply:
Thanks!